org.lbry.cli.json

```diff
@@ -0,0 +1,29 @@
+{
+  "id": "org.lbry.cli",
```

Comment:
`cli` isn't a proper application name.

Also CLI tools aren't a good candidate for flatpak.

Reply:
If CLI tools aren't good candidates for Flatpak, they shouldn't have `console-application` as component type, I guess.

Reply:
you are confusing AppStream and Flatpak. The former is an XDG spec, the latter just happen to use it. 

Also our submission documentation:
https://docs.flathub.org/docs/for-app-authors/requirements#impermissible-submissions

Reply:
What about this one?
https://flathub.org/en/apps/org.freedesktop.appstream.cli

Or?
https://flathub.org/en/apps/org.asamk.SignalCli